### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by 2004-okari**
In this mobile portfolio project:

   1. Created a contact section with help from coding patners
   2. Added form validation attributes
   3. Updated readme file to add features and contributors

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
# Hi @2004-okari 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that doesn't follow the proper GitHub flow. Please make sure the merge is from a 'feature' branch to a 'main' branch. In your case it should be from `contact_form` to `main`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+                <li class="list-1"><img src="images/Rectangle 57.svg" alt="rect" class="rec-image">Skills</li>
+                <li class="list">Database Management</li>
+                <li class="list">Version Control</li>
+                <li class="list">CLI</li>
+                <li class="list">App Design</li>
+                <li class="list">Web Development</li>
+            </ul>
+        </section>
+        <section class="section-5">
+            <form action="https://formspree.io/f/{form_id}" method="post" class="form-sec">
```
Comment (Shisui6) [inline index.html:229]: Your form currently doesn't integrate with the formspree service. Kindly update your action URL and replace `{form_id}` with the unique id given to you when you create a formspree account🙏

File: index.html [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+                <h3 class="contact-text">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h3>
+                <input type="type" name="name" placeholder="Full Name" required id="name-id" minlength="4" maxlength="30">
```
Comment (Shisui6) [inline index.html:233]: Kindly update the type attribute of the name input to be equal to `text` as it is a text input🙏

File: index.html [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+                <div class="seperate-line"></div>
+                <input type="email" name="email" placeholder="Email address" id="email-id" required>
+                <div class="seperate-line"></div>
+                <textarea name="text-area" placeholder="Write me something" id="text-area" rows="5" cols="30" maxlength="500"></textarea>
```
Comment (Shisui6) [inline index.html:237]: Kindly add the required attribute to your text area input field to ensure your users submit a message👍

File: index.html [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+                <input type="submit" name="Get in touch" id="submit" value="Get in touch" placeholder="Get in touch">
```
Comment (Shisui6) [inline index.html:238]: Kindly add button interaction styling to your buttons when hovered over and pressed. Please use the styles on Figma as a guide

image

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @2004-okari, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Added some form validations ✅
- Neat design ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @2004-okari , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Grean linters
✅ ... Form submission


---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**